CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
 # Changelog
 
+## 2021.8.9 (3 August 2021)
+
+### Enhancements
+
+1. Updated the preview to run by line in native notebooks. Set the `jupyter.experimental.debugging` setting to true, install ipykernel 6 on your selected kernel and a `Run by Line` button will appear on cell toolbars. Pressing it will start a lightweight debugging session and let you run the cell line by line.
+   ([#5607](https://github.com/microsoft/vscode-jupyter/issues/5607))
+
+### Fixes
+
+1. Fix for kernel not starting with correct path (causes DLL load and import modules failures).
```

Comment:
I believe this one was in the point release. I must have forgotten to merge the point release changelog back to main

Reply:
Yeah it looks like the point release changelog entries aren't there.

Reply:
Fixed